templates/components/form/header_content.html.twig

```diff
@@ -100,6 +100,16 @@
             {{ nametype }}
          {% endif %}
       </span>
+      {% if item.isField('is_dynamic') and item.fields['is_dynamic'] %}
+      <span {% if in_navheader %} class="mx-1 bg-secondary status rounded-1" title="{{ __('Is dynamic') }}"
```

Comment:
I'm not sure about changing the label/tooltip from "Automatic Inventory" to "Is dynamic". It seems like it would be more confusing.

Reply:
I agree to keep "Automatic inventory"

Reply:
agree too